src/main/java/org/skypro/skyshop/exceptions/ShopControllerAdvice.java

```diff
@@ -0,0 +1,17 @@
+package org.skypro.skyshop.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+
+@ControllerAdvice
```

Comment:
для RestController-ов надо использовать
```suggestion
@RestControllerAdvice
```

Reply:
Исправил